### DISCUSSION TITLE
SO 2.3 EOL

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#11547] [Unsupported Versions] [answered]
Hello!

When will be SO 2.3 EOL and will there be a migration path (at least for all my log data) or can I already plan to dump my distributed environment and setup everything from scratch? Thanks!

Cheers, Ben

**Comment by TOoSmOotH** [comment, marked as answer]
https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html

**Reply by ben-sec** [reply]
Thanks, I missed that one. However, what about custom configurations of my 2.3 installation (bpf configurations, rule settings, custom playbooks, dashboards, ...). Can I re-use some of them in 2.4 if I save them somewhere or do I have to start all over?

**Reply by Mav1814** [reply]
Hello!
This is the link for the migration from 2.3 to 2.4, you can find an overview of the process at 
[https://docs.securityonion.net/en/2.4/appendix.html](url)
